source/common/event/file_event_impl.cc

```diff
@@ -108,6 +109,13 @@ void FileEventImpl::assignEvents(uint32_t events, event_base* base) {
       this);
 }
 
+void FileEventImpl::updateEvents(uint32_t events) {
```

Comment:
Consider adding:
if (events == enabled_events_) {
  return;
}

Reply:
fixed in https://github.com/envoyproxy/envoy/pull/13787/commits/c5bb3d00c49aa1522e3c980ca7923ff258e8da1a